sympy/integrals/tests/test_prde.py

```diff
@@ -270,13 +327,3 @@ def test_parametric_log_deriv():
     assert parametric_log_deriv_heu(Poly(5*t**2 + t - 6, t), Poly(2*x*t**2, t),
     Poly(-1, t), Poly(x*t**2, t), DE) == \
         (2, 6, t*x**5)
-
-
-def test_issue_10798():
```

Comment:
As I could see the test I mentioned was already added by Kalevi (in a later PR for that issue). And here is this modified test in the right file (i.e `test_risch.py`) https://github.com/sympy/sympy/pull/12743/commits/616b0a0db0adcb9f3201b89c5f25cd6c933a187e#diff-72dfd0e04e99c9981ab3d8349b5eaa09R646